### ISSUE TITLE
Invalid colour contrast for default icons

### ISSUE THREAD

**Issue by JoelHaslam** [grafana/grafana#39791]
When presenting text, it is important to make sure that the contrast between text and background is sufficient enough to make the text readable. When adding in panels that containing statistics, Grafana offers default colourways to add extra context to these pieces of data.

**Steps to reproduce**:
I used the table that can (at least at the time of raising this) be found in [this dashboard](https://play.grafana.org/d/Fc6gwV4nz/transforms-config-from-query?orgId=1) located on play-grafana.
![image](https://user-images.githubusercontent.com/63413067/135288877-7531adf0-027d-4288-aeb2-87af7ad0c062.png)

The table shows the `color` selected, and the output that it would produce.

Using the app _**Accessibility insights for Windows**_, you can select the colour of the text and the colour of the background to calculate the contrast ratio.

**Actual Result**:
The required contrast ratios are 4.5:1 for regular text and 3:1 for large / bold text.

The _**red**_ option has a ratio of 3.268:1, but all other colour options have a contrast ratio of less than 3.

**Expected Result**:
Whether this text is considered regular or large/bold is slightly subjective, however I would expect to see a ratio of _at least_ 3 or greater for these elements.

**Relevant WCAG Criteria:** [1.4.3: contrast-minimum](https://www.w3.org/TR/WCAG21/#contrast-minimum)

**Environment**:
- Grafana version: 8.1.4
- Data source type & version: Default data used at [play.grafana.org](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1)
- User OS & Browser: Windows & Chrome
- Others:

**Comment by torkelo** [comment]
that is a demo that show  case that colors can from from the user data, so a user can specify whatever color they want in that demo. Grafana tries to optimise the text color as best it can based on the the user configured background color (should be maxed white here)

**Comment by JoelHaslam** [comment]
I'm struggling to understand this a little bit.

In the query for the dashboard linked above, it contains some CSV content:
```
Value, Color
OK, blue
Pretty bad, red
Yay it's green, green
Something is off, orange
No idea, #88AA00
Am I purple?, purple
```

I can change any of these colours to quite a wide range of options (cyan, maroon, navy as examples), where do the definitions for these colours come from? Where I think what I'd like to be able to do is define the colour `red` as a certain shade that has suitable colour contrasts.

I'm very new to Grafana so please bear with me while I try to get familiar with the interface 😆
